sites/partners/src/components/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -41,6 +55,16 @@ const FormUnits = ({ units, setUnits, disableUnitsAccordion, featureFlags }: Uni
   const { register, errors, clearErrors, getValues, control, setValue } = formMethods
   const listing = getValues()
 
+  const enableSection8Question = doJurisdictionsHaveFeatureFlagOn(
+    FeatureFlagEnum.enableSection8Question,
+    listing?.jurisdictions?.id
+  )
+
+  const enableUnitGroups = doJurisdictionsHaveFeatureFlagOn(
+    FeatureFlagEnum.enableUnitGroups,
+    listing?.jurisdictions?.id
+  )
```

Comment:
Issue: this is an issue with both the unit groups and section8 (and potentially other features as well). 

When you have access to more than jurisdiction and not all of them have the feature flag turned on the following happens:
* creating a listing from scratch and don't have a jurisdiction selected it is setting `enableUnitGroups` and `enableSection8Question` to true. Meaning that the fields are showing but they shouldn't be.

I believe it could be resolved by updating this call
```
  const enableUnitGroups = doJurisdictionsHaveFeatureFlagOn(
    FeatureFlagEnum.enableUnitGroups,
    listing?.jurisdictions?.id,
    !listing?.jurisdictions?.id
  )
```
If no jurisdiction is selected this function will only return true if all available jurisdictions have the feature.